.codegen/service.py.tmpl

```diff
@@ -275,7 +279,11 @@ self._api.do('{{.Verb}}', {{if .PathParts -}}
   {{- else if .Pagination -}}
     Iterator[{{.Pagination.Entity.PascalName}}]
   {{- else -}}
-    {{.Response.PascalName}}
+    {{- if .Response.IsExternal -}}
```

Comment:
can you do `{{template "type-nq" .Response}}`?

Reply:
If I do, then it yields the below diff. I believe we want to retain the current version where we use the listed type name.

```diff
--- a/databricks/sdk/service/settings.py
+++ b/databricks/sdk/service/settings.py
@@ -1164,7 +1164,7 @@ class WorkspaceConfAPI:
     def __init__(self, api_client):
         self._api = api_client

-    def get_status(self, keys: str, **kwargs) -> WorkspaceConf:
+    def get_status(self, keys: str, **kwargs) -> Dict[str, str]:
         """Check configuration status.

         Gets the configuration status for a workspace.
```